### ISSUE TITLE
Suspicious mappings for some genes

### ISSUE THREAD

**Issue by bgyori** [pyobo/pyobo#39]
Playing around with some mappings, I'm seeing some strange results, and it makes me wonder if mappings being to broader/narrower categories is the underlying issue. One example is hgnc:6880 (MAPK7) getting mapped to uniprot:Q02750 (MAP2K1.)
```
  "uniprot:Q02750": [
    {
      "provenance": "hgnc",
      "source": "hgnc:6880",
      "target": "eccode:2.7.11.24"
    },
    {
      "provenance": "https://github.com/sorgerlab/famplex/raw/master/equivalences.csv",
      "source": "eccode:2.7.11.24",
      "target": "fplx:MAPK"
    },
    {
      "provenance": "https://github.com/sorgerlab/famplex/raw/master/equivalences.csv",
      "source": "fplx:MAPK",
      "target": "eccode:2.7.12.2"
    },
    {
      "provenance": "hgnc",
      "source": "eccode:2.7.12.2",
      "target": "hgnc:6840"
    },
    {
      "provenance": "hgnc",
      "source": "hgnc:6840",
      "target": "uniprot:Q02750"
    }
  ],
```
Is this result expected by default, and it's up to the user to e.g., exclude mappings via eccode? Or is this something that should be fixed in the mappings generation?

**Comment by cthoyt** [comment]
hmm yes we will have to start curating rules about which mappings are false positives / should be is_a.

**Comment by bgyori** [comment]
In terms of the HGNC-EC relationships, EC codes are, as far as I understand always for classes of enzymes, so in general, the relationship should always be `isa`, not `xref`.

**Comment by cthoyt** [comment]
The more I think about it, the more I'm inclined to exclude mappings from genes to protein families like this. I would much rather have an ontological edge _gene product of_ (RO:0002204) connecting HGNC terms to UniProt terms, then allow uniprot terms to be connected to the EC code via an isA relationship

**Comment by cthoyt** [comment]
Should be fixed by #54
